### PR TITLE
add descriptions

### PR DESCRIPTION
**Pull request by mpeck**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -10,13 +11,15 @@ permissions:
 commands:
   component:
     executable: /usr/bin/cog-command
+    description: Enumerate and view status of defined statuspage.io components
```
Comment (christophermaier) [inline config.yaml]: I think we can leave the "statuspage.io" bit out of the command descriptions; I think it goes without saying 😄

**Comment by christophermaier** [comment]
I actually meant remove the entire "statuspage.io" string from the command descriptions, not just the ".io"

**Comment by mpeck** [comment]
eh, sorry. I just copied the description from the documentation. My brain is only at partial power today.

**Comment by christophermaier** [comment]
👍

**Pull request merged**